sound/soc/sof/ipc3-ops.h

```diff
@@ -16,6 +16,6 @@
 extern const struct ipc_pcm_ops ipc3_pcm_ops;
 extern const struct ipc_tplg_ops ipc3_tplg_ops;
 extern const struct ipc_tplg_control_ops tplg_ipc3_control_ops;
-extern const struct ipc_ops ipc3_ops;
+extern const struct sof_ipc_ops ipc3_ops;
```

Comment:
we probably need a fixup for all of ipc_pcm_ops, ipc_tplg_ops, ipc_tplg_controls_ops?